src/testFixtures/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/resource/model/CreateGoalRequestBuilder.kt

```diff
@@ -4,6 +4,7 @@ import java.time.LocalDate
 
 fun aValidCreateGoalRequest(
   title: String = "Improve communication skills",
+  // TODO RR-81 - set review date to null by default
   reviewDate: LocalDate = LocalDate.now().plusMonths(6),
```

Comment:
Should this arg be optional now? It can still have a default value if we think it makes sense, but now its optional?
Is that right?